pvlib/test/test_atmosphere.py

```diff
@@ -113,3 +114,40 @@ def test_first_solar_spectral_correction_supplied():
 def test_first_solar_spectral_correction_ambiguous():
     with pytest.raises(TypeError):
         atmosphere.first_solar_spectral_correction(1, 1)
+
+
+def test_kasten96_lt():
+    """Test Linke turbidity factor calculated from AOD, Pwat and AM"""
+    atmos_path = os.path.dirname(os.path.abspath(__file__))
+    pvlib_path = os.path.dirname(atmos_path)
+    melbourne_fl = tmy.readtmy3(os.path.join(
+        pvlib_path, 'data', '722040TYA.CSV')
```

Comment:
I'd prefer to define ~2-5 element arrays that span the range of allowed input values, and also define the expected return values. I can't quickly inspect the values that are being tested and expected in this test. Maybe these values span the space well enough?

```python
am = np.array([1, 3, 5])
pw = np.array([0, 2.5, 5])
aod700 = np.array([0, 0.1, 1])
lt_molineaux_expected = np.array('your precalculated values')
lt_bird_hulstrom_expected = np.array('your precalculated values')
...
np.assert_allclose(lt_molineaux_expected, lt_molineaux)
np.assert_allclose(lt_bird_hulstrom_expected, lt_bird_hulstrom)
```

Then you don't need to read a tmy file or lookup linke turbidity in the test. Seems more simple and more explicit to me. I'm not sure how much I trust the tmy and linke turbidity files anyways.